modules/ROOT/pages/design-add-api-dependency.adoc

```diff
@@ -109,5 +111,4 @@ After you add the rulesets as dependencies, you can view the conformance message
 == See Also
 
 * xref:api-governance::find-conformance-issues.adoc[Finding API Conformance Issues in API Specifications]
-* xref:exchange::to-create-an-asset.adoc[Create an Asset]
-
+* xref:exchange::to-create-an-asset.adoc[Create an Asset]
```

Comment:
Please, remember that we should use hardcoded text only if necessary. Is it necessary in this case?